src/database.ts

```diff
@@ -91,6 +112,55 @@ export const createDatabase = async (
         await collection.get(countryCode).push(userId).write();
       }
     },
+    addRemoteMember: async (userId, fromCountryCode, toCountryCode) => {
+      const collection = db.get('remoteIndex');
+      const currentState = collection.value();
+
+      if (currentState[userId] === undefined){
+        await collection.assign({
+          ...currentState,
+          [userId]: {from: fromCountryCode, to: toCountryCode}
+        })
+        .write()
+      }
+    },
+    hasRemoteMemberRegistered: (userId) => {
+      const collection = db.get('remoteIndex');
+      const members = collection.value() || {};
+
+      return Object.keys(members).includes(userId.toString())
+    },
+    getRemoteMembersFrom: (countryCode) => {
+      const collection = db.get('remoteIndex')
+      const members = collection.value() || [];
+
+      const filteredMembers = Object.entries(members).filter(([key, value]) => {
+        if (value["from"] !== undefined) {
+          return value["from"] === countryCode
+        }
+        return false
+      })
```

Comment:
this expression could be simplified to just be `return value["from"] === countryCode`

Reply:
Done.